shortlister/view.py

```diff
@@ -20,11 +20,20 @@ def view_applicant_details(self, applicant: Applicant):
         """Prints details of applicant to console."""
         print(f"Details for {applicant.name}: ")
         print(f"CV Path: {applicant.cv}")
-        print("Scores:")
-        if applicant.scores is not None:
+        print("Scores:", end=" ")
+
+        if applicant.scores == {}:
+            print("No scores")
+        else:
+            print()
             for criterion, score in applicant.scores.items():
                 print(f"{criterion.name:^20}: {score:^20}")
-        print(f"\nNotes: {applicant.notes}\n")
+
+        print("Notes:", end=" ")
```

Comment:
Comments above apply here too.

Reply:
Do you mean nothing as completely nothing? or:
```Notes:               ```

Reply:
I mean completely nothing. Don't need to handle it.